### ISSUE TITLE
Add. 14,667, foll. 72, 73 | https://bl.vuexistapps.us/manuscript/480 (Part19)

### ISSUE THREAD

**Issue by AndrewDraper6** [srophe/britishLibrary-data#956]
### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### Problems with Physical Description

- [ ] "They formed part of the fifteenth quire" seems to be extraneous.

### Categories of Physical Description Problems

- [ ] Whitespace
- [ ] Typo(s)
- [X] Incorrect content
- [ ] Missing content

**Comment by mikey-tang16** [comment]
Question @wlpotter 
"They formed part of the fifteenth quire" is indeed incomplete and taken out of the Contents. I'm not sure how to fix this one

**Comment by wlpotter** [comment]
Yeah we want to clarify that they were part of the fifteenth quire of the manuscript they were originally part of. Maybe something like the following?

>Two vellum leaves (Add. 14,667, foll. 72, 73), about 7 in. by 5 1/4 , much soiled and torn, which originally formed part of a fifteenth quire (ܝܗ).

**Comment by mikey-tang16** [comment]
updated accordingly
